packages/@o3r/localization/package.json

```diff
@@ -41,6 +41,12 @@
   "dependencies": {
     "tslib": "^2.4.1"
   },
+  "generatorDependencies": {
+    "@angular/cdk": "~15.1.2",
+    "@formatjs/intl-numberformat": "^8.0.2",
```

Comment:
No need to be in `generatorDependencies` as part of the `peerDeps`.
This is the only exception but it would make sense to be in peerDeps as well

Reply:
updated